### ISSUE TITLE
Bitcore wallet scam

### ISSUE THREAD

**Issue by sangeethasomanath18** [bitpay/bitcore-wallet-service#609]
I have installed bitcore wallet on my windows 10 laptop and there was one receiving address to receive my bitcoin.  I have externally sent my bitcoins to the bitcore wallet address and haven't received anything in the wallet for over a week.  Can someone help me out with this. I will lose all my bitcoin otherwise.

**Comment by matiu** [comment]
Was the transaction mined?

Can you share the address or transaction id?

**Comment by sangeethasomanath18** [comment]
No. It was sent from a local bitcoin wallet.  
1Gic3kN9wThLCzfpopNfWZ4F3k2y2qs8T
This was the address the bitcore wallet had.  The only one. And no bitcoins  have come to my wallet and it's been a week.

**Comment by isocolsky** [comment]
Since there is no activity associated with that address, I think you should check the source from where you sent the coins.

**Comment by TheGentleBunny** [comment]
I downloaded and install BitCore Wallet. I generated receiving address X4. All the while the wallet was synchronizing with a server. After 5 Days 24Hrs the wallet is still only up to 50 %. It's zapping my CPU power, and still not ready to recievce payment? Piece of Sh*t... what will happen if my funds are sent before the bloody wallet is synced. FYI I use an Electrum wallet on my 32 bit pc and its working strait away. Dam Bitcore there was no warning
